packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -61,7 +71,8 @@ class RemoteConfigManager extends EventEmitter {
       cached_target_files: [] // updated by `parseConfig()`
     }
 
-    this.appliedConfigs = new Map()
+    this._handlers = new Map()
+    const appliedConfigs = this.appliedConfigs = new Map()
```

Comment:
wow, i know it gets hoisted but, use before declaration is weeeeeird, we can just move it above this.clients ?